internal/command/instance.go

```diff
@@ -102,6 +102,7 @@ type InstanceSetup struct {
 	}
 	LockoutPolicy struct {
 		MaxAttempts              uint64
```

Comment:
Should we rename `MaxAttempts` to `MaxPasswordAttempts` here?

Reply:
Hmm that would mean a breaking change in the config since this is directly exposed for the DefaultInstance and FirstInstance configs. The question is if we're ok with that?

Reply:
I haven't noticed it came from the config. I just thought you forgot to rename that one.

We already have breaking config changes this release with the machine secret config move. IMO one more doesn't matter, we need to be notifying people already anyway. I leave it up to you what you want to do here :)

Reply:
you're right. changed it and added a note in the PR description